jstz_cli/src/sandbox/daemon.rs

```diff
@@ -119,12 +119,10 @@ fn init_client(cfg: &Config) -> Result<()> {
     Ok(())
 }
 
-fn client_bake(cfg: &Config) -> Result<()> {
-    OctezClient::bake(
-        cfg,
-        &cfg.jstz_path.join("logs/client.log"),
-        &["for", "--minimal-timestamp"],
-    )?;
+fn client_bake(cfg: &Config, log_file: &File) -> Result<()> {
+    // SAFETY: When a baking fails, then we want to silently ignore the error and
```

Comment:
Question: what are the reasons while baking can fail?

Reply:
The node is down, or inconsistent context, or many other (undocumented) reasons 😅 😩